### ISSUE TITLE
Formula calculate without stop

### ISSUE THREAD

**Issue by scutcs2015** [cunnane/xloil#111]
I have developed an asynchronous UDF that requests data from an interface，like this
```
async def _getUrlImpl(x):
    async with aiohttp.ClientSession() as session:
        async with session.post("http://localhost:5000/test", json={"value": x}) as response:
            data = await response.json()
            return data['data']


@xloil.func(local=False, rtd=True)
async def test(x):
    yield await _getUrlImpl(x)
```
When the formula in cell is `=test(5)`，it's ok ，it will return result after the request finish. But when the formula in cell is `=test(test(5))`, it will calculate without stop,  send two request to the url each time. 
How can I solve this?

**Comment by cunnane** [comment]
This should be fixed in v0.19.0 which I just released.  Multiple RTD functions in a cell is definitely a tricky area so let me know if you see any issues.

**Comment by scutcs2015** [comment]
> This should be fixed in v0.19.0 which I just released. Multiple RTD functions in a cell is definitely a tricky area so let me know if you see any issues.

Good！The new version has effectively resolved my issue
